src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php

```diff
@@ -80,17 +80,22 @@ public function action(Group $pools, Cache $cache, Database $dbForConsole, Regis
                     $dbForProject->setDefaultDatabase('appwrite');
                     $dbForProject->setNamespace('_' . $project->getInternalId());
                     $collectionsCreated = $dbForProject->count(Database::METADATA);
-                    $message = ' (' . $collectionsCreated . ') collections where found on project (' . $project->getId() . '))';
-                    if ($collectionsCreated < (count($collectionsConfig) + 2)) {
-                        Console::error($message);
+                    if ($collectionsCreated === 0) {
+                        if ($commit === true) {
```

Comment:
This can just be $commit only, right?

```suggestion
                        if ($commit) {
```